### ISSUE TITLE
Catapault should be Catapult.

### ISSUE THREAD

**Issue by flipdazed** [flipdazed/SoftwareDevelopment#6]
**Bugged Version** 0_0_0
**Lines** 356

```
2 * [Card('Catapault', (7, 0), 6)],
```

**Issue**
Spelling

**Fix**
Spell correctly!

**Comment by flipdazed** [comment]
```
        2 * [Card('Catapult', (7, 0), 6)],
```

**Comment by flipdazed** [comment]
This issue caused me to think the new game and replay parameters were different.

**Comment by flipdazed** [comment]
line `219` in `engine.py` shows the error

```
        2 * [Card('Catapault', (7, 0), 6)],
```

Shows the advantage in using a settings file.

**Comment by flipdazed** [comment]
closed see #70
